maths/volume.dart

```diff
@@ -0,0 +1,100 @@
+import 'dart:math' show pi;
+
+import 'package:test/test.dart';
+
+/* 
+ ** Program to calculate the volume of the shapes
+ ** Reference: https://www.cuemath.com/measurement/volume
+*/
+
+/* Calculate the volume for a Cuboid */
+double volCuboid(double l, double b, double h) {
+  return l * b * h;
+}
+
+/* Calculate the volume for a Cube */
+double volCube(double a) {
+  return power(a, 3);
+}
+
+/* Calculate the volume for a Cone */
+double volCone(double r, double h) {
+  return pi * power(r, 2) * h / 3;
+}
+
+/* Calculate the volume for a Cylinder */
+double volCylinder(double r, double h) {
+  return pi * power(r, 2) * h;
+}
+
+/* Calculate the volume for a Sphere  */
+double volSphere(double r) {
+  return pi * power(r, 3) * 4 / 3;
+}
+
+/* Calculate the volume for a Hemisphere */
+double volHemisphere(double r) {
+  return pi * power(r, 3) * 2 / 3;
+}
+
+/* Calculate the volume for a Pyramid */
+double volPyramid(double baseLength, double baseWidth, double height) {
+  return baseLength * baseWidth * height / 3;
+}
+
+/* Calculate the volume for a Triangular Prism */
+double volTriangularPrism(double baseLengthTriangle, double heightTriangle, double height) {
+  return (baseLengthTriangle * heightTriangle * height) / 2;
+}
+
+/* Calculate the volume for a Pentagonal Prism */
+double volPentagonalPrism(double pentagonalLength, double pentagonalBaseLength, double height) {
```

Comment:
I think it would be better to rename the functions with complete words 
like `volumeOfCuboid` || `volumeOfCube`.